CMakeLists.txt

```
@@ -5,6 +5,11 @@ project(spdlog_vendor)
 find_package(ament_cmake REQUIRED)
 ament_add_default_options()
 
+# Avoid DOWNLOAD_EXTRACT_TIMESTAMP warning for CMake >= 3.24
+if (POLICY CMP0135)
```

```suggestion
if(POLICY CMP0135)
```

(this is why CI is failing)